assets/src/edit-story/components/form/switch.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+const SwitchContainer = styled.div`
+  appearance: none;
+  position: relative;
+  background: ${({ theme }) => theme.colors.fg.v1};
+  border-radius: 100px;
+  color: ${({ theme }) => theme.colors.mg.v3};
+  font-family: ${({ theme }) => theme.fonts.body2.family};
+  font-size: ${({ theme }) => theme.fonts.body2.size};
+  line-height: ${({ theme }) => theme.fonts.body2.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.body2.letterSpacing};
+  padding: 8px 3px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  border: none;
+`;
+
+const RadioButton = styled.input.attrs(({ checked, value, id }) => ({
+  type: 'radio',
+  name: 'switch',
+  checked,
+  value,
+  id,
+}))`
+  overflow: hidden;
+  min-width: unset !important;
+  width: 1px !important;
+  height: 1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  margin: -1px !important;
```

Comment:
What's with all this `!important`? To circumvent WP styling?

Reply:
Related: https://github.com/google/web-stories-wp/issues/349

Reply:
Yes, to replace original wp styling. Any ideas to avoid these? @spacedmonkey @barklund

Reply:
I'd have to take a closer look. We can do that in #349.